anta/cli/nrfu/utils.py

```diff
@@ -34,15 +34,19 @@ def print_settings(context: click.Context, report_template: Optional[pathlib.Pat
     console.print(Panel.fit(message, style="cyan", title="[green]Settings"))
 
 
-def print_table(results: ResultManager, device: Optional[str] = None, test: Optional[str] = None) -> None:
+def print_table(results: ResultManager, device: Optional[str] = None, test: Optional[str] = None, group_by: Optional[str] = None) -> None:
     """Print result in a table"""
     reporter = ReportTable()
-    if device:
-        console.print(reporter.report_summary_hosts(result_manager=results, host=device))
-    if test:
-        console.print(reporter.report_summary_tests(result_manager=results, testcase=test))
-    if device is None and test is None:
+    if device or test:
+        message = "The test and device options are not supported yet."
+        console.print(Panel.fit(message, style="red", title="[red]Warning"))
```

Comment:
The lines that implements these options are being deleted by this PR, let's keep these lines that implements the simple use case: filter one device and/or one test.